sources/api/netdog/src/wicked/mod.rs

```diff
@@ -3,16 +3,28 @@
 //!
 //! The structures in this module are meant to be created from the user-facing structures in the
 //! `net_config` module.  `Default` implementations for WickedInterface exist here as well.
+mod bonding;
 mod dhcp;
 mod static_address;
+mod vlan;
 
 use crate::interface_name::InterfaceName;
+use crate::net_config::devices::bonding::{BondMonitoringConfig, NetBondV1};
+use crate::net_config::devices::interface::NetInterfaceV2;
+use crate::net_config::devices::vlan::NetVlanV1;
+use crate::net_config::devices::NetworkDeviceV1;
+use crate::wicked::bonding::{
+    WickedArpMonitoringConfig, WickedBondMode, WickedMiiMonitoringConfig,
+};
+pub(crate) use bonding::WickedBond;
 pub(crate) use dhcp::{WickedDhcp4, WickedDhcp6};
+pub(crate) use error::Error;
 use serde::Serialize;
 use snafu::ResultExt;
 pub(crate) use static_address::{WickedRoutes, WickedStaticAddress};
 use std::fs;
 use std::path::Path;
+pub(crate) use vlan::WickedVlanTag;
```

Comment:
Does this need to be `pub(crate)`?  I don't think it's used anywhere else.

Reply:
I don't think it is anymore. I'll confirm and fix.